app/src/pages/guestbook/[id]/edit.tsx

```diff
@@ -23,27 +23,28 @@ export default function GuestBookEdit() {
     } = useGuestBookMessage(router.query.id as string);
     const detailsHref = `/guestbook/${message?.messageId}`;
 
-    const [draftMessage, setDraftMessage] = useState<IGuestBookMessage>();
-    const isLoading = isMessageLoading || isUndefined(draftMessage);
+    // const [draftMessage, setDraftMessage] = useState<IGuestBookMessage>();
+    // const isLoading = isMessageLoading || isUndefined(draftMessage);
+    const isLoading = isMessageLoading;
 
     // `message` is the unaltered message originally fetched from the API
     // `draftMessage` is the message object being modified by the user
     // When `message` loads for the first time, store it in `draftMessage
-    useEffect(() => {
-        if (!isUndefined(message) && isUndefined(draftMessage)) {
-            setDraftMessage(message);
-        }
-    }, [message, draftMessage]);
+    // useEffect(() => {
+    //     if (!isUndefined(message) && isUndefined(draftMessage)) {
+    //         setDraftMessage(message);
+    //     }
+    // }, [message, draftMessage]);
```

Comment:
Can you delete all unused code now? It's confusing to have commented-out code, because it's not clear why it's there (is it broken? is it no longer needed? is it needed but not ready yet?)

Reply:
it's no longer needed. I can remove it